pkg/operation/botanist/cleanup.go

```diff
@@ -215,19 +214,3 @@ func (b *Botanist) CleanKubernetesResources(ctx context.Context) error {
 		cleanResourceFn(c, &corev1.PersistentVolumeClaimList{}, false, PersistentVolumeClaimCleanOptions),
 	)(ctx)
 }
-
-// DeleteOrphanEtcdMainPVC delete the orphan PVC associated with old etcd-main statefulsets as a result of migration in Release 0.22.0 (https://github.com/gardener/gardener/releases/tag/0.22.0).
-func (b *Botanist) DeleteOrphanEtcdMainPVC(ctx context.Context) error {
```

Comment:
/cc @swapnilgm FYI, the migration code here is already removed.

@stoyanr I might have told you to remove it, don't remember fully, however, now I think it might be better to remove the migration code as part of a separate PR. Can you please adapt the PR such that this function (as well as the task in the flow executing it) is kept?

@swapnilgm I'm thinking about removing this migration code with `0.27.0` (so, not immediately with the next 0.26.0 release).

Reply:
Is this absolutely needed? I would avoid making such changes to the PR if possible, as it has been extensively tested as it is.

Reply:
IMHO you can safely reintroduce this function (+task in the flow) without affecting any test results. All your changes are not related to this independent task.